services/API-service/src/api/admin-area/admin-area.controller.ts

```diff
@@ -121,15 +144,14 @@ export class AdminAreaController {
   @ApiQuery({ name: 'placeCodeParent', required: false, type: 'string' })
   @ApiResponse({
     status: 200,
-    description:
-      '(Relevant) admin-areas boundaries and attributes for given country, disater-type and lead-time',
+    description: 'Event admin areas GeoJSON FeatureCollection',
   })
   @Get(':countryCodeISO3/:disasterType/:adminLevel')
-  public async getAdminAreas(
-    @Param() params,
-    @Query() query,
+  public async getEventAdminAreas(
```

Comment:
This naming is not fully correct, because this also gets admin-areas in no-event scenario.